dataproc/dataproc_quickstart/quickstart.go

```diff
@@ -0,0 +1,194 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This sample shows how you can use the Cloud Dataproc Client library to create a
+// Cloud Dataproc cluster
+
+// [START dataproc_quickstart]
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"time"
+
+	dataproc "cloud.google.com/go/dataproc/apiv1"
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/option"
+	dataprocpb "google.golang.org/genproto/googleapis/cloud/dataproc/v1"
+)
+
+func quickstart(w io.Writer, projectID, region, clusterName, jobFilePath string) error {
+	ctx := context.Background()
+
+	// Create the cluster client
+	endpoint := fmt.Sprintf("%s-dataproc.googleapis.com:443", region)
+	clusterClient, err := dataproc.NewClusterControllerClient(ctx, option.WithEndpoint(endpoint))
+	if err != nil {
+		return fmt.Errorf("Error creating the cluster client: %s", err)
```

Comment:
Minor nit: Generally you don't capitalize error messages since they tend to be concatenated and/or wrapped in other error messages.

Reply:
Fixed, thanks